modules/installation-osp-about-kuryr.adoc

```diff
@@ -40,7 +40,9 @@ OpenShift SDN over an OpenStack network.
 Conversely, Kuryr is not recommended in the following cases:
 
 * You use provider networks or tenant VLANs.
-* Your deployment uses many Services on a few hypervisors. Each OpenShift
-  Service creates an Octavia Amphora virtual machine in OpenStack that hosts a
-  required load balancer.
-* UDP Services are needed.
+* In {rh-openstack} versions earlier than 16, or if the {rh-openstack} Neutron
+  ML2/plugin is not OVN or ovn-octavia Octavia driver is not enabled, if:
```

Comment:
Perhaps might be good to replace the last `if` with `and your deployment uses:` and update the following line accordingly. Just to reinforce that the next points are the main concern when combined with version<16 / not ovn / not ovn-octavia.